sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -284,6 +322,8 @@ def run_pipeline(self, pipeline, options):
     pipeline.visit(DataflowRunner.group_by_key_input_visitor())
     self._bundle_repeat = self._bundle_repeat or options.view_as(
         pipeline_options.DirectOptions).direct_runner_bundle_repeat
+    FnApiRunner._num_workers = max(FnApiRunner._num_workers, options.view_as(
```

Comment:
Rather than max, it's probably better to have one take precedence. Alternatively, only have one way to set this (e.g. via the option).

Reply:
Changed to support option only.